src/main/java/org/opensearch/knn/index/query/KNNWeight.java

```diff
@@ -213,8 +214,8 @@ private Map<Integer, Float> doANNSearch(final LeafReaderContext context, final B
         String modelId = fieldInfo.getAttribute(MODEL_ID);
         if (modelId != null) {
             ModelMetadata modelMetadata = modelDao.getMetadata(modelId);
-            if (modelMetadata == null) {
-                throw new RuntimeException("Model \"" + modelId + "\" does not exist.");
+            if (!ModelUtil.isModelCreated(modelMetadata)) {
+                throw new RuntimeException("Model \"" + modelId + "\" is not created.");
```

Comment:
nit: please use String.format for string concatenation

Reply:
@martin-gaievski this comes up a couple times because we do not do it for a lot of existing code. Do you know if its possible to add something to spotless to automatically fix this?

Reply:
it should be possible, we did some custom template for spotless in neural-search, please check this PR https://github.com/opensearch-project/neural-search/pull/515

Reply:
@martin-gaievski is there a comment for it or file? Im not seeing it in formatting